### DISCUSSION TITLE
POM / Delrin / Acetal Filament Print Profile Setting

### DISCUSSION THREAD

**Discussion by dwisetiyadi** [OrcaSlicer/OrcaSlicer#5236] [General]
I need to print with POM / Delrin / Acetal material, but can not find the profile for it. Any people has experience print this filament with Orca Slicer?

**Comment by TomSeals** [comment]
Welp going on 2 years and no response... You ever continue going down this path? Ive heard drywall joint tape is a great surface to use. Watched someone use glue stick to put it on their bed and then put gluestick on top and it seemed to print pretty well on it.
